style/tables.css

```diff
@@ -4,7 +4,7 @@
 .ProseMirror table {
   border-collapse: collapse;
   table-layout: fixed;
-  width: 100%;
+  /* width: 100%; */
```

Comment:
You forget to uncomment here : )

Also, could you run `pnpm run fix` to format the code?

Reply:
😬, fixed